genai-perf/tests/test_config_command.py

```diff
@@ -47,6 +49,24 @@ def test_model_name(self):
         config = ConfigCommand(user_config)
 
         self.assertEqual(config.model_names, ["gpt2"])
+        self.assertEqual(config.get_field("model_names").required, True)
+
+    def test_multi_model_name(self):
+        """
+        Test that a configuration with multiple model names is parsed correctly
+        """
+        # yapf: disable
+        yaml_str = ("""
+            model_names:
+                - gpt2
```

Comment:
if your system has 2 instance of a model, how would we handle that here? do we care? do we have to list the model with 2 instances twice?

Reply:
Number of instances of a model is a "model parameter" and shouldn't be relevant here. I don't think we care in GAP/PA